src/components/vehicleDataGroup/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Box, List, useMultiStyleConfig } from '@chakra-ui/react';
```

Comment:
The same here about using our components instead importing them from `@chakra-ui/react`